SAKPaaS/src/app/components/location-panel/location-panel.component.scss

```
@@ -50,3 +49,42 @@ h1 {
 .sectionheading {
   margin-top: 30px;
 }
+
+.searchMinimized {
+
+}
+
+.minimized {
+  top: calc(100vh - 120px) !important;
```

Is this `!important` really necessary? Please plase this in `@media` since this introduces trouble if window is scaled up after minimizing.
```suggestion
@media (min-width: 600px) {
.minimized {
  top: calc(100vh - 120px) !important;
}
}
```